app/src/androidTest/java/org/akvo/flow/TempTestFileFactory.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (C) 2010-2017 Stichting Akvo (Akvo Foundation)
```

Comment:
This copyright header is outdated, Stellan has made some changes to it and it now looks like in this file:
https://github.com/akvo/akvo-flow-mobile/blob/develop/app/src/main/java/org/akvo/flow/service/ApkUpdateService.java